tox.ini

```
@@ -1,43 +1,65 @@
 [tox]
 envlist=
-    py{37,38,39,310,311,pypy3}
-    lint
+    py{38,39,310,311}-core
+    py{38,39,310,311}-lint
+    py{38,39,310,311}-wheel
+    py311-wheel-windows
+    docs
 
 [flake8]
-max-line-length= 100
-exclude= tests/*
-
-[isort]
-combine_as_imports=True
-force_sort_within_sections=True
-include_trailing_comma=True
-known_standard_library=pytest
-known_first_party=eth_bloom
-line_length=21
-multi_line_output=3
-use_parentheses=True
+exclude=venv*,.tox,docs,build
+extend-ignore=E203
+max-line-length=88
+per-file-ignores=__init__.py:F401
 
 [testenv]
 usedevelop=True
 commands=
-    pytest tests
-extras=
-    test
-deps =
-    eth-hash[pycryptodome]
-basepython =
-    py37: python3.7
+    core: pytest {posargs:tests/core}
```

Same issue we saw recently here. There are tests that exist outside of `tests/core` within `tests`. This is only testing that the library can be imported at the moment. Let's change this to the below perhaps:

```suggestion
    core: pytest {posargs:tests}
```

I'm also seeing a warning that might be nice to silence in the core tests related to `python_paths` within `pytest.ini`. Let's remove that line from that file.

---

Another oversight - `pytest.ini` should be deleted, as all the config has moved to `pyproject.toml`. It is now deleted.